docs/docs/quick-start.md

```diff
@@ -1,66 +1,66 @@
 ---
-title: Quick Start
+title: Démarrage rapide
 ---
 
-This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
+Cette page de démarrage rapide est dédié aux développeurs d'un niveau intermédiaire à avancé. Pour une intro plus facile à Gatsby, [dirigez-vous vers notre tutoriel](/tutorial/)!
 
-## Use the Gatsby CLI
+## Utiliser le Gatbsy CLI
 
 <EggheadEmbed
   lessonLink="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line"
-  lessonTitle="Quick Start with Gatsby: Create, Develop, and Build Gatsby Sites From the Command Line"
+  lessonTitle="Démarrage rapide avec Gatsby : Créer, Développer et Builder les sites Gatbsy en ligne de commande"
 />
 
-**Note**: this video uses `npx`, which is a tool to execute an npm package without first installing it. Running the command `npx gatsby new` is the same as running `gatsby new` after installing the gatsby-cli on your computer.
+**Note**: Cette vidéo utilise `npx`, qui est un outil pour éxecuter un package npm sans avoir à l'installer d'abord. Lancer la commande `npx gatsby new` est la même chose que lancer `gatsby new` après avoir installé le gatsby-cli sur votre ordinateur.
 
-### Install the Gatsby CLI.
+### Installer le Gatsby CLI.
 
 ```shell
 npm install -g gatsby-cli
 ```
 
-### Create a new site.
+### Créer un nouveau site
 
 ```shell
 gatsby new gatsby-site
 ```
 
-### Change directories into site folder.
+### Changer les répertoires dans le dossier du site.
 
 ```shell
 cd gatsby-site
 ```
 
-### Start development server.
+### Démarrer le serveur de développement.
 
 ```shell
 gatsby develop
 ```
 
-Gatsby will start a hot-reloading development environment accessible by default at `localhost:8000`.
+Gatsby va démarrer un environnement de réchargement à chaud accessible par défaut à l'adresse `localhost:8000`.
```

Comment:
We loose some informations here e. g. « development server » is not mentioned. 

More generally I’m wondering if a French dev will easily understand what is a « environnement de rechargement à chaud »? What do you think?

Reply:
Good question, actually most French Devs use English terminologies like hot-reloading.
Translating such words directly to French would confuse them the more.

Reply:
How about "rechargement immédiat" if it must be translated?

Reply:
I like this last version ;)

Reply:
Parcel uses "Remplacement de module à chaud" for its docs https://parceljs.org/hmr.html. Would you think this would be more consistent? From my perspective, both are very clear.

Reply:
Both are very clear indeed. I think we'll have to make a decision quickly as this md file is stuck. "Remplacement de module à chaud" is more relevant for webpack and parcel in my opinion.